client/my-sites/people/team-list/team.jsx

```diff
@@ -0,0 +1,151 @@
+		};
+	},
+
+	isLastPage() {
+		return this.props.totalUsers <= this.props.users.length + this.props.excludedUsers.length;
+	},
+
+	fetchHasCompleted() {
+		return this.props.fetchInitialized && ! this.props.fetchingUsers;
+	},
+
+	render: function() {
+		var key = deterministicStringify( omit( this.props.fetchOptions, [ 'number', 'offset' ] ) ),
+			headerText = this.translate( 'Team', { context: 'A navigation label.' } ),
+			listClass = ( this.state.bulkEditing ) ? 'bulk-editing' : null,
+			people,
+			showRoles = ! this.props.search && config.isEnabled( 'manage/people/role-filtering' );
+
+		if ( this.fetchHasCompleted() && ! this.props.users.length && this.props.fetchOptions.search ) {
+			return (
+				<NoResults
+					image="/calypso/images/people/mystery-person.svg"
+					text={
+						this.translate( 'No results found for {{em}}%(searchTerm)s{{/em}}',
+							{
+								args: { searchTerm: this.props.search },
+								components: { em: <em /> }
+							}
+						)
+					} />
+			);
+		}
+
+		if ( this.fetchHasCompleted() && this.props.site && ( this.props.users.length || this.props.role ) ) {
+			if ( this.props.search && this.props.totalUsers ) {
+				headerText = this.translate(
+					'%(numberPeople)d Person Matching {{em}}"%(searchTerm)s"{{/em}}',
+					'%(numberPeople)d People Matching {{em}}"%(searchTerm)s"{{/em}}',
+					{
+						count: this.props.users.length,
+						args: {
+							numberPeople: this.props.totalUsers,
+							searchTerm: this.props.search
+						},
+						components: {
+							em: <em />
+						}
+					}
+				);
+			}
+
+			if ( this.props.role && this.props.users.length === 0 ) {
+				const noResultsMessage = this.translate(
+					'No users have this role on this site'
+				);
+				people = (
+					<NoResults
+						image="/calypso/images/people/mystery-person.svg"
+						text={ noResultsMessage }
+					/>
+				);
+			} else {
+				people = (
+					<InfiniteList
+						key={ key }
+						items={ this.props.users }
+						className="people-selector__infinite-list"
+						ref="infiniteList"
+						fetchingNextPage={ this.props.fetchingUsers }
+						lastPage={ this.isLastPage() }
+						fetchNextPage={ this._fetchNextPage }
+						getItemRef={ this._getPersonRef }
+						renderLoadingPlaceholders={ this._renderLoadingPeople }
+						renderItem={ this._renderPerson }
+						guessedItemHeight={ 126 }>
+					</InfiniteList>
+				);
+			}
+		} else {
+			people = this._renderLoadingPeople();
+		}
+
+		return (
+			<div style={{minHeight: "300px"}}>
+				<PeopleListSectionHeader
+					role={ this.props.role }
+					showRoles={ showRoles }
+					label={ headerText }
+					site={ this.props.site }
+					count={ this.props.fetchingUsers || this.props.fetchOptions.search ? null : this.props.totalUsers } />
+				<Card className={ listClass }>
+					{ people }
+				</Card>
+				{ this.isLastPage() && <div className="infinite-scroll-end" /> }
+			</div>
+		);
+	},
+
+	_renderPerson: function( user ) {
```

Comment:
we try and avoid using underscore-leading names. `renderPerson` is fine. the same goes for the other `_`-prefixed functions